mythril/analysis/modules/state_change_external_calls.py

```diff
@@ -66,12 +69,15 @@ def get_issue(
         logging.debug(
             "[EXTERNAL_CALLS] Detected state changes at addresses: {}".format(address)
         )
-        description_head = (
-            "The contract account state is changed after an external call. "
-        )
+        read_or_write = "write"
+        if global_state.get_current_instruction()["opcode"] == "SLOAD":
+            read_or_write = "read"
+        address_type = "user defined" if self.user_defined_address else "fixed"
+        description_head = "Persistent state {} after call".format(read_or_write)
```

Comment:
@JoranHonig, @wuestholz is this the description used by harvey?

Reply:
https://github.com/ConsenSys/harvey/blob/b05a643c5e4288115ff1d8842d651b9e8ccd6320/extensions/warnings.go#L397

Yup.

Reply:
Lol, I don't have the permissions for the repo :laughing: